homeassistant/components/stream/core.py

```diff
@@ -35,6 +36,19 @@
 PROVIDERS: Registry[str, type[StreamOutput]] = Registry()
 
 
+class Orientation(IntEnum):
+    """Orientations for stream transforms."""
```

Comment:
Based on the comment for `TRANSFORM_MATRIX_TOP`, these are based on Exif orientation tags? Should that be noted here?

Reply:
Let me adjust the comments.